main/entry.c

```diff
@@ -156,7 +156,7 @@ extern const char *tagFileName (void)
 
 extern void abort_if_ferror(MIO *const mio)
 {
-	if (mio_error (mio))
+	if (mio != NULL && mio_error (mio))
```

Comment:
These NULL checks are here because we bypass opening the MIO which is always NULL in our case.

Reply:
We have to recondier the output stream. More abstracted interfaces may be needed. Temporary, I use this code.